### ISSUE TITLE
[Serious] Active user interface component lacks 3 to 1 contrast ratio

### ISSUE THREAD

**Issue by ntallman** [APTrust/registry#105]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87723 |
| **Checkpoint** | 1.4 Distinguishable › Non-Text Contrast - Active User Interface Components (1.4.11.a) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 06 Alerts |
| **Test Case Name** | AP Trust Registry (Demo) |
| **URL** | <https://demo.aptrust.org/alerts> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Product** | Website |
| **Date Reported** | 05/11/26 at 12:17 pm |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | a3caccess+@gmail.com |

## Description

The visual boundary of an active user interface component lacks 3 to 1 contrast ratio.
The Mark Read/Unread gold/gray envelope icon on white background has a low contrast, making it difficult to see.

## Recommendation

Rule : 
The visual characteristics that indicate the component's purpose or type (e.g. is it a checkbox, text field, slider, switch, etc.) must have sufficient contrast of 3 to 1 with the adjacent background. Exceptions exist.

How To Fix: 
Fix this issue by adjusting the user interface component boundary and/or background to increase the contrast with either the inner or outer adjacent background to at least 3 to 1.

Background: 
People who have low vision or are colorblind may have difficulty perceiving that an element is interactive or what its state is - e.g. whether it has keyboard focus, is selected/checked/pressed, etc. - if the contrast between the element boundaries and/or state indicators and its background or adjacent colors is insufficient. When an interactive element's visual boundaries and state indicators have adequate contrast, people who have low vision or are colorblind are more likely to be able to perceive which elements on the screen are interactive and what their current state is.

## Source Code

```
<span data-name="alertReadIcon" class="material-icons is-grey" aria-hidden="true">drafts</span>
<span class="is-sr-only">Mark as Unread</span>
```

## Screenshots

![06_user interface contrast1](https://raw.githubusercontent.com/APTrust/registry/master/docs/accessibility-audit/screenshots/87723/06_user_interface_contrast1.png)
![06_user interface contrast](https://raw.githubusercontent.com/APTrust/registry/master/docs/accessibility-audit/screenshots/87723/06_user_interface_contrast.png)
